pkg/ccl/streamingccl/logical/logical_replication_writer_processor.go

```diff
@@ -828,7 +828,7 @@ func makeInsertQuery(fqTableName string, td catalog.TableDescriptor) string {
 	originTSIdx := argIdx
 	baseQuery := `
 INSERT INTO %s (%s, crdb_internal_origin_timestamp)
-VALUES (%s, %d)
+VALUES (%s, $%d)
```

Comment:
This change LGTM.  Let's get this in so that it doesn't block testing.  Longer term, it would be nice to validate LWW semantics with a unit test.